File: src/sample/StartUp.java
package sample;

import javafx.application.Application;
import javafx.scene.Scene;
import javafx.stage.Stage;

public class StartUp extends Application {

    @Override
    public void start(Stage primaryStage) throws Exception {
        MainScreen mainScreen = new MainScreen();
        Logic logic = new Logic(mainScreen);
        Engine engine = new Engine(mainScreen, logic);

        Scene scene = new Scene(mainScreen, 800, 600);

        primaryStage.setTitle("GeneticT1");
        primaryStage.setScene(scene);
        primaryStage.setMaximized(true);
        primaryStage.show();
    }
}

File: src/sample/StartUp.kt
package sample

import javafx.application.Application
import javafx.scene.Scene
import javafx.stage.Stage

class StartUp : Application() {

  @Throws(Exception::class)
  override fun start(primaryStage: Stage) {
    val mainScreen = MainScreen()
    val logic = Logic(mainScreen)
    Engine(mainScreen, logic)

    val scene = Scene(mainScreen, 800.0, 600.0)

    primaryStage.title = "GeneticT1"
    primaryStage.scene = scene
    primaryStage.isMaximized = true
    primaryStage.show()
  }
}
